### DISCUSSION TITLE
Enable SyslistView32 (syslistview32-enabler) Mod: Custom Process Exclusion List has no effect. :-(

### DISCUSSION THREAD

**Discussion by FLskydiver** [ramensoftware/windhawk-mods#665] [General]
So I really love this mod for how more efficient it is with space in TILES, LIST, and DETAILS views .... but for two issues:

The first* is that when viewing a photo gallery (using either of the MEDIUM, LARGE, or EXTRA LARGE ICON views), the space-savings is drastically reversed.  The icons have _far_ too much padding, and very few of them can be seen clearly at one time.

*(The second I will create a separate post for).

I hoped to make this a non-issue by taking advantage of the Advanced Tab's feature that purportedly allows one to exclude specific paths from being altered from the default Windows 10 display settings.  Alas, I can't make this work.  

I even tried using the "excluded" field in the registry when "custom excluded" failed to have an effect ... but this also did nothing.

Preferably, I'd like to see the Icon View padding tightened up to the Windows 10 standard.  Failing the possibility of that, I'd hope this issue can be fixed (either by fixing the "custom exclude" feature — or better explaining it in case I'm simply doing something wron0)g.  

Thank you!

**Comment by m417z** [comment]
Tagging @Anixx, the mod author.
